plugins/feedback/src/components/CreateFeedbackModal/CreateFeedbackModal.tsx

```diff
@@ -0,0 +1,322 @@
+                :&nbsp;
+                {feedbackType === 'BUG'
+                  ? issueTags.map(issueTitle => (
+                      <Chip
+                        key={issueTitle.toLowerCase()}
+                        clickable
+                        variant={
+                          selectedTag === issueTitle ? 'default' : 'outlined'
+                        }
+                        color="secondary"
+                        onClick={() => handleChipSlection(issueTitle)}
+                        label={issueTitle}
+                      />
+                    ))
+                  : feedbackTags.map(feedbackTitle => (
+                      <Chip
+                        key={feedbackTitle.toLowerCase()}
+                        clickable
+                        variant={
+                          selectedTag === feedbackTitle ? 'default' : 'outlined'
+                        }
+                        color="primary"
+                        onClick={() => handleChipSlection(feedbackTitle)}
+                        label={feedbackTitle}
+                      />
+                    ))}
+              </Typography>
+            </Grid>
+            <Grid item xs={12}>
+              <TextField
+                autoComplete="off"
+                id="summary"
+                label="Summary"
+                value={summary.value}
+                variant="outlined"
+                placeholder="Enter Summary"
+                onChange={handleInputChange}
+                onBlur={handleValidation}
+                error={summary.error}
+                helperText={
+                  summary.error
+                    ? summary.errorMessage
+                    : `${summary.value.length}/${summaryLimit}`
+                }
+                required
+              />
+            </Grid>
+
+            <Grid item xs={12}>
+              <TextField
+                inputMode="text"
+                autoComplete="off"
+                id="description"
+                label="Description"
+                value={description.value}
+                variant="outlined"
+                placeholder="Enter description"
+                multiline
+                minRows={6}
+                maxRows={10}
+                onChange={handleInputChange}
+              />
+            </Grid>
+          </Grid>
+        </DialogContent>
+        <DialogActions className={classes.actions}>
+          <Button onClick={props.handleModalCloseFn}>Cancel</Button>
+          <Button
+            onClick={handleSubmitClick}
+            color={feedbackType === 'FEEDBACK' ? 'primary' : 'secondary'}
+            variant="contained"
+            disabled={summary.error || summary.value.length === 0}
+          >
+            {feedbackType === 'FEEDBACK' ? 'Send Feedback' : 'Report Bug'}
+          </Button>
+        </DialogActions>
+        {props.serverType.toLowerCase() !== 'mail' &&
```

Comment:
This is called twice, I think it'd be better to just call it once when the component is initialized.

Reply:
Also I'd recommend using a ternery instead for the conditional.

Reply:
Did the necessary changes